pkgs/desktops/gnome/core/mutter/default.nix

```diff
@@ -63,6 +64,11 @@ let self = stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       inherit zenity;
     })
+
+    (fetchpatch {
+      url = "https://salsa.debian.org/gnome-team/mutter/-/raw/91d9bdafd5d624fe1f40f4be48663014830eee78/debian/patches/x11-Add-support-for-fractional-scaling-using-Randr.patch";
+      sha256 = "m6PKjVxhGVuzsMBVA82UyJ6Cb1s6SMI0eRooa+F2MY8=";
+    })
```

Comment:
```suggestion

    # https://salsa.debian.org/gnome-team/mutter/-/blob/ubuntu/master/debian/patches/x11-Add-support-for-fractional-scaling-using-Randr.patch
    (fetchpatch {
      url = "https://salsa.debian.org/gnome-team/mutter/-/raw/91d9bdafd5d624fe1f40f4be48663014830eee78/debian/patches/x11-Add-support-for-fractional-scaling-using-Randr.patch";
      sha256 = "m6PKjVxhGVuzsMBVA82UyJ6Cb1s6SMI0eRooa+F2MY8=";
    })
```

Reply:
This makes it easier to update since the patch is in the ubuntu/master branch and not the debian/master branch